runtimes/scala/play-framework-2.md

```diff
@@ -65,47 +45,31 @@ So for an application using the PostgreSQL add-on, you can set:
 ```bash
 db.default.driver=org.postgresql.Driver
 db.default.url="jdbc:postgresql://"${POSTGRESQL_ADDON_HOST}"/"${POSTGRESQL_ADDON_DB}
-db.default.user=${POSTGRESQL_ADDON_USER}
+db.default.username=${POSTGRESQL_ADDON_USER}
 db.default.password=${POSTGRESQL_ADDON_PASSWORD}
 ```
 
-## Known problems with Play! 2
-
-Please read the following if your project fails with this error:  
-
-`sbt.ResolveException: unresolved dependency: play#sbt-plugin;2.0: not found`
+## Custom config file
 
-Some versions of Play2 try to retrieve a nonexistent version of
-"sbt-plugin" which is required by the framework to work.
-You have two options to fix this problem:
-
-You can set the "play.version" environment variable in the
-`/clevercloud/sbt.json` file.  
-For example, for Play 2.0.4:
+If you don't want to use the default `conf/application.conf` configuration
+file, you can use the `SBT_DEPLOY_GOAL` environment variable:
 
-``` javascript
-{
-  "deploy": {
-    "goal": "-Dplay.version=2.0.4"
-  }
-}
+```
+SBT_DEPLOY_GOAL=-Dconfig.resource=clevercloud.conf
 ```
 
-Otherwise, you can modify `plugins.sbt` in the project folder of your
-app like the following:
+## HTTPS support
 
-``` scala
-// Comment to get more information during initialization
-logLevel := Level.Warn
+HTTPS is handled by Clever Cloud ahead of your application, your application
+retrieves the traffic in plain http. To be able to use `request.secure`, you
+have to add `trustxforwarded=true` in `application.conf`. If you need an
```

Comment:
That was before Play 2.4, the blog post linked after actually explains how to do it with Play 2.4 and play 2.5 changed slightly again: https://gist.github.com/urcadox/a54c20f0b86f1e9d36a341c861efd2dd